rofication/_queue.py

```diff
@@ -30,80 +36,105 @@ def __iter__(self) -> Iterator[Notification]:
     def lock(self) -> threading.Lock:
         return self._lock
 
-    def save(self, filename: str) -> None:
+    def save(self, filename: str = None) -> None:
+        if not filename:
+            filename = self._queue_file
+
+        if not filename:
+            warn("Unable to save notification queue, no file specified")
```

Comment:
Will this emit in syslog or somewhere else user visible?

Reply:
I'm using warn there as that's what _queue.py already had for other messages. In any case, its output just go to stderr.